parsers/mainz.py

```diff
@@ -36,68 +36,70 @@
-    # Source: https://www.studierendenwerk-mainz.de/essentrinken/speiseplan/
-    '1': 'mit Farbstoff',
-    '2': 'mit Konservierungsstoff',
-    '3': 'mit Antioxidationsmittel',
-    '4': 'mit Geschmacksverstärker',
-    '5': 'geschwefelt',
-    '6': 'geschwärzt',
-    '7': 'gewachst',
-    '8': 'Phosphat',
-    '9': 'mit Süßungsmitteln',
-    '10': 'enthält eine Phenylalaninquelle',
-    'S' : 'Schweinefleisch',
-    'G' : 'Geflügelfleisch',
-    'R' : 'Rindfleisch',
-    'Gl' : 'Gluten',
-    'We' : 'Weizen (inkl. Dinkel)',
-    'Ro' : 'Roggen',
-    'Ge' : 'Gerste',
-    'Haf': 'Hafer',
-    'Kr' : 'Krebstiere und Krebstiererzeugnisse',
-    'Ei' : 'Eier und Eiererzeugnisse',
-    'Fi' : 'Fisch und Fischerzeugnisse',
-    'En' : 'Erdnüsse und Erdnusserzeugnisse',
-    'So' : 'Soja und Sojaerzeugnisse',
-    'La' : 'Milch und Milcherzeugnisse',
-    'Sl' : 'Sellerie und Sellerieerzeugnisse',
-    'Sf' : 'Senf und Senferzeugnisse',
-    'Se' : 'Sesamsamen und Sesamsamenerzeugnisse',
-    'Sw' : 'Schwefeldioxid und Sulfite > 10mg/kg',
-    'Lu' : 'Lupine und Lupinerzeugnisse',
-    'Wt' : 'Weichtiere und Weichtiererzeugnisse',
-    'Nu' : 'Schalenfrüchte',
-    'Man': 'Mandel',
-    'Has': 'Haselnüsse',
-    'Wa' : 'Walnüsse',
-    'Ka' : 'Kaschunüsse',
-    'Pe' : 'Pecanüsse',
-    'Pa' : 'Paranüsse',
-    'Pi' : 'Pistatien',
-    'Mac': 'Macadamianüsse',
-    'icon:S.png' : 'Scheinefleisch',
-    'icon:R.png' : 'Rindfleisch',
-    'icon:Fi.png' : 'Fisch',
-    'icon:Gl.png' : 'Glutenfrei',
-    'icon:La.png' : 'Lactosefrei',
-    'icon:Vegan.png' : 'Vegan',
-    'icon:Veggi.png' : 'Vegetarisch'
-    
+  # Source: https://www.studierendenwerk-mainz.de/essen-trinken/speiseplan
+  '1': 'mit Farbstoff',
+  '2': 'mit Konservierungsstoff',
+  '3': 'mit Antioxidationsmittel',
+  '4': 'mit Geschmacksverstärker',
+  '5': 'geschwefelt',
+  '6': 'geschwärzt',
+  '7': 'gewachst',
+  '8': 'Phosphat',
+  '9': 'mit Süßungsmitteln',
+  '10': 'enthält eine Phenylalaninquelle',
+  'S' : 'Schweinefleisch',
+  'G' : 'Geflügelfleisch',
+  'R' : 'Rindfleisch',
+  'Gl' : 'Gluten',
+  'We' : 'Weizen (inkl. Dinkel)',
+  'Ro' : 'Roggen',
+  'Ge' : 'Gerste',
+  'Haf': 'Hafer',
+  'Kr' : 'Krebstiere und Krebstiererzeugnisse',
+  'Ei' : 'Eier und Eiererzeugnisse',
+  'Fi' : 'Fisch und Fischerzeugnisse',
+  'En' : 'Erdnüsse und Erdnusserzeugnisse',
+  'So' : 'Soja und Sojaerzeugnisse',
+  'La' : 'Milch und Milcherzeugnisse',
+  'Sl' : 'Sellerie und Sellerieerzeugnisse',
+  'Sf' : 'Senf und Senferzeugnisse',
+  'Se' : 'Sesamsamen und Sesamsamenerzeugnisse',
+  'Sw' : 'Schwefeldioxid und Sulfite > 10mg/kg',
+  'Lu' : 'Lupine und Lupinerzeugnisse',
+  'Wt' : 'Weichtiere und Weichtiererzeugnisse',
+  'Nu' : 'Schalenfrüchte',
+  'Man': 'Mandel',
+  'Has': 'Haselnüsse',
+  'Wa' : 'Walnüsse',
+  'Ka' : 'Kaschunüsse',
+  'Pe' : 'Pecanüsse',
+  'Pa' : 'Paranüsse',
+  'Pi' : 'Pistatien',
+  'Mac': 'Macadamianüsse',
+}
+
+iconLegend = {
+  'icon:S.png' : 'Scheinefleisch',
```

Comment:
I think these now need to lose the `icon:` prefix, since in line 114 you look up just the `basename`.

Reply:
Oh, that was quick. 😂